ccontainermain.go

```diff
@@ -621,7 +623,7 @@ func main() {
 	if exeStop != "" {
 		_, err := stopExtraService(exeStop)
 		if err != nil {
-			log.Printf("\nError shutting down eXtra service: %s; err: \n", exeStop, err)
+			log.Printf("\nError shutting down eXtra service: %s; err: %s\n", exeStop, err)
```

Comment:
I'm not sure of this one @daimor 
as the method already has:
if err := c.Run(); err != nil {
 		errMsg := out.String()
 		log.Printf("Error in stopping service(s) %s:", xstop)
 		log.Printf("Err: %s; %s", errMsg, err)
 		os.Exit(1)
 	}
--

Please comment...
thanks